### DISCUSSION TITLE
Certificate error when opening AiO interface via admin settings

### DISCUSSION THREAD

**Discussion by RotasOpera** [nextcloud/all-in-one#2291] [Questions] [answered]
### Steps to reproduce
1. Open the Nextcloud AiO interface via the Administration settings

### Expected behavior 
The AiO management interface should open with port 8443
### Actual behavior 
Error message due to self signed certificate because the link in settings points to port 8080

### Host OS 
Debian 11

#### Nextcloud AIO version 
v4.8.1
#### Current channel 
beta
#### Other valuable info 
I just set this up for the third time, everything works as expected. It's just annoying that the link points to an incorrect port. I do not have a reverse proxy in front of the apache container.

![image](https://user-images.githubusercontent.com/35852251/230425439-2f6db412-1abf-44ff-9cfb-3c9ed1a542bb.png)

![image](https://user-images.githubusercontent.com/35852251/230424476-0f02aef9-08b7-4d84-be38-d3a1d12ca581.png)

**Comment by szaimen** [comment, marked as answer]
Hi, here is described how you can change this: https://github.com/nextcloud/all-in-one#how-to-easily-log-in-to-the-aio-interface

**Reply by RotasOpera** [reply]
Oh boy, I feel incredibly dumb. Thanks for the hint
